### PR TITLE
Fixed #591

### LINKED ISSUE #591: Issue with vendor wise commission
If we set vendor wise commission as 0 and set global commission then the vendor commission gets calculated on global commission. 
But if the vendor wise commission is set something other than 0 then the commission is calculated as per that.

### PR DESCRIPTION
**Pull request by user-promi**

### CODE REVIEW

**Comment by Sangi0** [comment]
https://watch.screencastify.com/v/zBpzDow9sc1p5r0lYn9q

File: classes/class-mvx-calculate-commission.php [hunk 1]
```diff
@@ -742,7 +742,7 @@ public function get_commission_amount($product_id = 0, $vendor_id = 0, $variatio
                             return array('commission_val' => $category_wise_commission);
                         }
                         $vendor_commission = get_user_meta($vendor->id, '_vendor_commission', true);
-                        if ($vendor_commission) {
+                        if ($vendor_commission != '') {
```
Comment (moumitahalder) [inline classes/class-mvx-calculate-commission.php]: why like ds? any functions exist or not?

**Review by moumitahalder — CHANGES_REQUESTED** [review]

**Comment by user-promi** [comment]
I didn't use the empty function because 0 can come in the variable, so i used the strlen function

File: classes/class-mvx-calculate-commission.php [hunk 2]
```diff
@@ -742,7 +742,7 @@ public function get_commission_amount($product_id = 0, $vendor_id = 0, $variatio
                             return array('commission_val' => $category_wise_commission);
                         }
                         $vendor_commission = get_user_meta($vendor->id, '_vendor_commission', true);
-                        if ($vendor_commission != '') {
+                        if (strlen($vendor_commission) > 0) {
```
Comment (moumitahalder) [inline classes/class-mvx-calculate-commission.php:745]: is $vendor_commission return string?

**Review by moumitahalder — CHANGES_REQUESTED** [review]

**Pull request merged**